runtime/src/accounts_db.rs

```diff
@@ -13450,153 +13429,6 @@ pub mod tests {
         assert_eq!(0, next_candidates.len());
     }
 
-    #[test]
-    fn test_select_candidates_by_total_usage_3_way_split_condition() {
```

Comment:
test no longer respresents a valid use-case as it requires multiple stores per slot.